agent/agentssh/agentssh.go

```diff
@@ -346,6 +348,12 @@ func (s *Server) startPTYSession(session ptySession, magicTypeLabel string, cmd
 	session.DisablePTYEmulation()
 
 	if !isQuietLogin(session.RawCommand()) {
```

Comment:
I think we should relax the constraint here since a service banner seems more important than a MOTD. Perhaps the presence of `.hushlogin` shouldn't silence it. The login shell check should be kept though.

Reply:
Agreed!